5주차/13459/13459_cpp_이지영.cpp

```diff
@@ -0,0 +1,91 @@
+#include <iostream>
+#include <algorithm>
+#include <cstring>
+#include <cmath>
+#include <vector>
+using namespace std;
+
+int N, M;
+char A[10][10]; //���� ���� �Է� ��� char��
+bool endd = false;
```

Comment:
전역변수는 default 값이 false입니다.
```suggestion
bool endd;
```